addons/mail/static/tests/message/message_tests.js

```diff
@@ -672,6 +673,24 @@ QUnit.test("should not be able to reply to temporary/transient messages", async
     await contains(".o-mail-Message [title='Reply']", { count: 0 });
 });
 
+QUnit.test("squashed transient message should not have date in the sidebar", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "Channel 1", channel_type: "chat" });
+    const { openDiscuss } = await start();
+    openDiscuss(channelId);
+    await insertText(".o-mail-Composer-input", "/who");
+    await click(".o-mail-Composer-send:enabled");
+    await contains(".o-mail-Message");
+
+    await insertText(".o-mail-Composer-input", "/who", { replace: true });
+    await click(".o-mail-Composer-send:enabled");
+    await click(".o-mail-Message.o-squashed");
```

Comment:
Useless empty newline, I'd drop it.
When  looking at tests at a glance, it's easier to spot LOCs of a test when tests are separated by empty newlines and there are none in them.